cli/script/command-executor.ts

```diff
@@ -166,14 +167,27 @@ function appAdd(command: cli.IAppAddCommand): Promise<void> {
         });
 }
 
+function getOwnerEmail(map: CollaboratorMap): string {
+    var ownerEmail: string = "";
+    if (map) {
+        Object.keys(map).forEach((email: string) => {
+            if ((<CollaboratorProperties>map[email]).permission === "Owner") {
+                ownerEmail = email;
+            }
+        });
```

Comment:
Looks like `.find()` is not a valid method on the arrays, will use `.filter()` instead.